Add profile page for SSS (#1)
security-summer-school/.github: profile/README.md

<p align="center">
  <img src="https://security-summer-school.github.io/SSS-logo.png" width="215" alt="Security Summer School" />
</p>
<p align="center">
  <img src="https://img.shields.io/github/contributors/security-summer-school/binary" />
    <a href="https://bit.ly/DiscordSecuritySummerSchool">
    <img src="https://img.shields.io/discord/715162845305634816.svg?label=discord&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2" />
  </a>
</p>

> ### Security Summer School is a security workshop held by people passionate about security, professors, PhD, MSci, BSc students from University POLITEHNICA of Bucharest and industry partners Ixia, a Keysight Business.

Reviewer: - This message is too long for a `###` tag. Try simple bold or nothing at all
- Add GRX as a partner so @raresvis doesn't get sad

Reviewer 2: Also PenTest tools as partner.

Author: Done.

> 

Reviewer: ```suggestion
```

## Projects

### :bulb: Learn the most known vulnerabilities

> From zero-to-hero the four week event holds lessons and workshops two or three times a week on vulnerabilities and how to identify and get advantage from them.

Reviewer: ```suggestion
**From zero to hero:** the four-week event holds lessons and workshops two or three times a week on vulnerabilities and how to identify and take advantage of them.
```

### :mag_right: Hands-on learning

> At the end of each presentation, you will have to solve problems, practicing the presented vulnerabilities.

Reviewer: ```suggestion
At the end of each presentation, you will have to solve challenges showcasing the presented vulnerabilities.
```

These will run as a CTF where you should find the flag and demonstrate your power.

> ### :trophy: CTF & Prizes!

Reviewer: ```suggestion
### :trophy: CTFs & Prizes!
```

Two CTFs will be organized, one in the middle of the summer school and the other at the end of it.

> Those who get involved will also receive prizes.

Reviewer 2: Remove this. We don't have winner prizes anymore. We're in communism, since @teodutu 's involvement, and everyone gets something.

### :busts_in_silhouette: Security Essentials

Perfect for the ones who want to step into security fundamentals.

### :rocket: Web Security

> Web is already everywhere.

Reviewer: ```suggestion
The Web is already everywhere.
```

> Web is already everywhere. Start before it’s too late.

Reviewer: ```suggestion
Web is already everywhere.
Start before it’s too late.
```

### :key: Binary Security

> Do you want to break all things in a binary? This is the place.

Reviewer: ```suggestion
Do you want to break all things in a binary?
This is the place.
```

## Community  

Security Summer School is an open source project that anyone in the community can use, improve, and enjoy.
We'd love you to join us!
Here's a few ways to find out what's happening and get involved.

### Learn and Connect

Using or want to use Security Summer School? Find out more here:

- [Discord](https://bit.ly/DiscordSecuritySummerSchool): Join the discussion with Security Summer School community!
- [Facebook](https://www.facebook.com/SSSUPB/): Follow us on Facebook to get the latest news!
- [Youtube](https://bit.ly/YoutubeSecuritySummerSchool): Follow us on Youtube to get access to all workshops!

### Develop and Contribute

> If you want to get more involved by contributing to Security Summer School, join us here:

Reviewer 3: ```suggestion
If you want to contribute to Security Summer School, join us here:
```

> If you want to contribute to Security Summer School, join us here:
> 
> - [Github](https://github.com/security-summer-school): Development takes place here!

Reviewer: ```suggestion
If you want to contribute to Security Summer School, join us on [Github](https://github.com/security-summer-school): development takes place here!
```

> You can find out how to contribute to these docs in our [Contribution Guidelines](https://security-summer-school.github.io/docs/contribution-guidelines/).

Reviewer 3: If we don't really have some guidelines at the moment, we could remove this line.
Alternatively, we could add a blank page containing a `TODO`/`Under construction` message to remind us that we need to work on it in the future.

Author: I think we should have a contribution guideline, I opened an issue [here] to track the progress on it.
[here]: https://github.com/security-summer-school/security-summer-school.github.io/issues/11
